### PR TITLE
UI changes for the sample app.

### PR DESCRIPTION
**Pull request by ryanwilson**
Moved the buttons out of the list, added a navigation title, and made
Identity have optional properties since it's more appropriate (vs an
empty string).

### CODE REVIEW

**Comment by ryanwilson** [comment]
# Before
Before Empty
Before Filled

# After
After Empty
After Filled

File: Example/Messaging/Sample/Sample/ContentView.swift [hunk 1]
```diff
@@ -123,15 +140,35 @@ struct DetailView: View {
...
-      Text("Token: \(self.identity.token)")
+      Text("InstanceID: \(self.identity.instanceID ?? "None")")
+      Text("Token: \(self.identity.token ?? "None")")
     }
   }
 }
 
 struct ContentView_Previews: PreviewProvider {
+  // A fake filled identity for testing rendering of a filled cell.
+  static let filledIdentity: Identity = {
```
Comment (chliangGoogle) [inline Example/Messaging/Sample/Sample/ContentView.swift:151]: oh this is really nice cuz I found rendering sometimes breaks for trying to get the real identity data

File: Example/Messaging/Sample/Sample/ContentView.swift [hunk 2]
```diff
@@ -123,15 +140,35 @@ struct DetailView: View {
...
+  }()
+
   static var previews: some View {
-    ContentView().environmentObject(Identity())
+    Group {
+      ContentView().environmentObject(Identity())
+
+      ContentView()
+        .environmentObject(filledIdentity)
+//        .environment(\.sizeCategory, .accessibilityLarge)
```
Comment (chliangGoogle) [inline Example/Messaging/Sample/Sample/ContentView.swift:169]: should we remove the comment line

File: Example/Messaging/Sample/Sample/ContentView.swift [hunk 3]
```diff
@@ -144,5 +181,7 @@ struct IdentityButtonStyle: ButtonStyle {
       .background(LinearGradient(gradient: Gradient(colors: [Color.blue, Color.pink]),
                                  startPoint: .leading, endPoint: .trailing))
       .cornerRadius(40)
+      // Push the button down a bit when it's pressed.
+      .scaleEffect(configuration.isPressed ? 0.9 : 1)
```
Comment (chliangGoogle) [inline Example/Messaging/Sample/Sample/ContentView.swift:185]: nice!

File: Example/Messaging/Sample/Sample/ContentView.swift (hunk 1 above)
Comment (ryanwilson) [reply Example/Messaging/Sample/Sample/ContentView.swift:151]: Yeah it's really helpful being able to inject data like this :D

File: Example/Messaging/Sample/Sample/ContentView.swift (hunk 2 above)
Comment (ryanwilson) [reply Example/Messaging/Sample/Sample/ContentView.swift]: Good call, done.

**Pull request merged**